sim/validator.py

```diff
@@ -116,6 +116,34 @@ def pipeline(
         shutil.rmtree(btb_seq_backup_path)
         shutil.rmtree(results_path)
 
+def benchmark(genomes_path, results_path, output_path):
+    # Validate input
+    if not os.path.exists(results_path):
+        raise Exception("Results path does not exist: ", results_path)
+    if not os.path.exists(genomes_path):
+        raise Exception("Genomes path does not exist: ", genomes_path)
+    if not os.path.exists(output_path):
+        raise Exception("Output path does not exist: ", output_path)
+    genomes = genome.from_directory(genomes_path)
+    # TODO: handle when glob does not return a unique path
+    path = glob.glob(results_path + '/Results_*')[0] + '/'
```

Comment:
This is no longer required. It's handled in `sequenced.from_results_dir(path)`

Reply:
As far as i can tell we still need to append `'/Results_*'` to `results_path`.

Do you just mean validating that it `path` contains the btb-seq output?